README.md

```diff
@@ -49,8 +49,14 @@ SkyPilot supports your existing GPU, TPU, and CPU workloads, with no code change
 
 Install with pip or [from source](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html):
 ```
-pip install "skypilot[aws,gcp,azure,lambda,ibm,scp]"  # choose your clouds
+pip install "skypilot[aws,gcp,azure,lambda,ibm,scp,oci]"  # choose your clouds
 ```
+<p align="center">
```

Comment:
Looks great! We can have the image link to https://skypilot.readthedocs.io/en/latest/getting-started/installation.html?